### DISCUSSION TITLE
How to get DRM working?

### DISCUSSION THREAD

**Discussion by 4KLife** [AppleBreak1/EP45-UD3P-Customac#7] [Q&A] [answered]
Hello, I followed your note and have RX570 + Q8300 working but no DRM in Safari. How can to fix?

**Comment by AppleBreak1** [comment, marked as answer]
Hi,

Regarding the possible fixes for streaming DRM protected contents, it's best to refer to [WhateverGreen](https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.Chart.md) manual and try different configurations. 


Suggestion, 

If you haven't done already, change SMBIOS to iMacPro1,1 and add either of the below boot-arg or dgpu property depending on the version of macOS you are on.


For Catalina or Older

- As in boot-arg: shikigva=80 
 
- As in device property: shikigva |number| 80


For Big Sur and Newer

- unfairgva=1 or unfairgva |number| 1

**Reply by 4KLife** [reply]
It's working! Red screen is no more when streaming prime videos. Merry Christmas to you sir!
